hippo.go

```diff
@@ -126,80 +143,128 @@ func (c *Client) Do(ctx context.Context, req *http.Request) ([]byte, error) {
 	return buf, nil
 }
 
-func (c *Client) EncodeTagBatchPart(rFull *TagBatchPart) ([][]byte, int, error) {
+// SendBatchPart sends a batch to the server, in multiple requests, if necessary.
+// - may return non-nil SendBatchResponse on error, it'll report how far we got
```

Comment:
do you mean `SendBatchResult`?

Reply:
https://github.com/kentik/gohippo/pull/9/commits/392dd6fa99eb7fbfca436c07f6f8dce57f01f90e#diff-6de703128b8cdd5f7cf717b969ac848cdb6220e788e80ca80e52f6e6ad47dba7

👍